docs/metadata.yml

```diff
@@ -0,0 +1,11 @@
+kind: LearningResources
+name: subs-related-docs
```

Comment:
@bluemorpho-mpw I believe we have a doc with this name already: https://github.com/RedHatInsights/quickstarts/blob/main/docs/quickstarts/subs-related-docs/subs-related-docs.yaml#L4

Is that related? Should we be overriding the content of that tile instead?

Reply:
I was trying to add a new tile with the links added to the ticket.